test.js

```diff
@@ -0,0 +1,3 @@
+  foo.bar()
```

Comment:
__[development][eslint]__
* Error - Expected indentation of 0 tabs but found 2 spaces. (indent)
* Error - 'foo' is not defined. (no-undef)
* Error - Missing semicolon. (semi)

<!-- Comment by [SideCI](https://sideci.com) -->